package.json

```diff
@@ -149,5 +150,8 @@
     "commitizen": {
       "path": "./node_modules/cz-conventional-changelog"
     }
+  },
+  "installConfig": {
+    "pnp": true
```

Comment:
Just to clarify in case someone else reads this: I have a PR that makes Yarn work under PnP, but it requires to bump Jest to 23+, which currently doesn't support Node 4. Since our test environment still supports Node 4 (not for long), I can't merge it yet.